homeassistant/components/somfy/__init__.py

```diff
@@ -156,13 +157,8 @@ def has_capability(self, capability):
 @Throttle(SCAN_INTERVAL)
 async def update_all_devices(hass):
     """Update all the devices."""
-    from requests import HTTPError
-    from oauthlib.oauth2 import TokenExpiredError
-
     try:
         data = hass.data[DOMAIN]
         data[DEVICES] = await hass.async_add_executor_job(data[API].get_devices)
-    except TokenExpiredError:
-        _LOGGER.warning("Cannot update devices due to expired token")
-    except HTTPError:
-        _LOGGER.warning("Cannot update devices")
+    except HTTPError as err:
+        _LOGGER.warning("Cannot update devices: %s", err.response.status_code)
```

Comment:
You can use fstring now to ease the code.

Reply:
I don't believe we've landed on using f-strings in log statements just yet; the above is the currently-accepted-and-utilized pattern.

Reply:
Based on the logging level set by the application, logging statements can be ignored. If we would use f-strings, we are doing work to create a new string that might be thrown away. PyLint will actually check to make sure we use the logging format.